### DISCUSSION TITLE
Use in my project

### DISCUSSION THREAD

**Discussion by jumpyjava** [schwabe/ics-openvpn#1745] [General]
Hello! How can I use ics-openvpn in my project in android studio? I tried importing it as a module, but I can't use it in classes.

**Comment by schwabe** [comment]
This is a standalone app. Trying to import it into another app is not going to work.

**Reply by jumpyjava** [reply]
Why? I'm currently looking at projects on Github, they use ics-openvpn as a library. I need to use OpenVpn in my project, can you help me?

**Reply by schwabe** [reply]
Ask those people please what they have done. This project is NOT a library. Other people might have made changes to use a library but they have also not contributed those changes back. Also ensure that you understand that using part of my code makes your program GPL2/3 and open source.

**Comment by davixdedem** [comment]
> Hello! How can I use ics-openvpn in my project in android studio? I tried importing it as a module, but I can't use it in classes.

Hi mate, I came up with a project called [Pi Starlink](https://github.com/davixdedem/Pi-Starlink). The app works with the APIs imported by OpenVPN for Android. The source is open, so feel free to check it out and get inspiration for your own project.

https://github.com/davixdedem/Pi-Starlink
